src/Api/GuestCarts.php

```diff
@@ -205,7 +205,7 @@ public function removeItem($cartId, $itemId)
      * @return Response
      * @throws Exception
      */
-    public function quoteLink($cartId, $adminEmail, $customerEmail, $customerFirstName, $customerLastName, $saleRep, $salesLocation, $salesChannel = "Email", $orderType = 'standard'): Response
+    public function quoteLink($cartId, $adminEmail, $customerEmail, $customerFirstName, $customerLastName, $saleRep, $salesLocation, $salesChannel = "Email",$archyProjectId = null, $orderType = 'standard'): Response
```

Comment:
What happens here if we the code on M2 is not ready to receive the new fields we add? I am wondering if this is backward compatible?

Reply:
@Medalink, 
We will start receive 500 error, both should release together.

Reply:
No changes requested but I am curious about some things:
* why did you add `$archyProjectId` to the middle of the function argument list instead of at the end?
  * no change needed if it doesn't cause backward compatibility issues for y'all
* ~why add this method to the `GuestCarts` API class instead of creating a new class?~
  * never mind; git history gave me the answer to this one

Reply:
As far as I can tell in M2 the param was added in the middle so this is matching that. Not sure why it was not bolted onto the end tho in M2. /shrug